courses/templates/courses/partials/after_one_year_body.html

```diff
@@ -0,0 +1,106 @@
+            {{ course_details.continuation_stats.continuing_or_complete }}%
+        </p>
+
+        <p>
+            of students are still on the course or have completed it
+        </p>
+    </div>
+</div>
+
+<div class="discover-uni-container">
+    <div class="one-year__block">
+        <div class="one-year__qualification-charts">
+            <div class="one-year__chart discover-uni-chart bar" data-value="{{course_details.continuation_stats.continuing}}">
+                {% include 'courses/partials/bar_chart.html' %}
+                <label class="one-year-block__chart-label">Are still on the course</label>
+            </div>
+
+            <div class="one-year__chart discover-uni-chart bar" data-value="{{course_details.continuation_stats.gained}}">
+                {% include 'courses/partials/bar_chart.html' %}
+                <label class="one-year-block__chart-label">Have completed the course (started in the final year due to prior study)</label>
+            </div>
+
+            <div class="one-year__chart discover-uni-chart bar" data-value="{{course_details.continuation_stats.dormant}}">
+                {% include 'courses/partials/bar_chart.html' %}
+                <label class="one-year-block__chart-label">Are taking a break from their studies</label>
+            </div>
+
+            <div class="one-year__chart discover-uni-chart bar" data-value="{{course_details.continuation_stats.lower}}">
+                {% include 'courses/partials/bar_chart.html' %}
+                <label class="one-year-block__chart-label">Have left with a lower qualification</label>
+            </div>
+
+            <div class="one-year__chart discover-uni-chart bar" data-value="{{course_details.continuation_stats.left}}">
+                {% include 'courses/partials/bar_chart.html' %}
+                <label class="one-year-block__chart-label">Have left without a qualification</label>
+            </div>
+
+            <div class="chart-explanation">
+                <div class="chart-explanation__link">
+                    <img class="information-icon" src="{% static 'images/information.svg' %}" alt="">
+                    <p class="information-link">Chart labels explained</p>
+                </div>
+
+                <div class="chart-label-explanation">
+                    <h4 class="chart-label-explanation__heading">
+                        Chart labels explained
+                        <span class="chart-label-explanation__close">X</span>
+                    </h4>
+
+                    <div class="chart-label-explanation__body">
+                        <h5 class="chart-label-explanation__sub-heading">
+                            Continue at the university or college
+                        </h5>
+
+                        <p>
+                            Students who continue to study after the first year of their course, either on the same course or changing to another course at the same university or college.
```

Comment:
Does this have to be hardcoded? Just thinking in the future, OfS may want to change the wording?